deploy/kubernetes/overlays/dev/WARNING.md

```diff
@@ -0,0 +1,8 @@
+WARNING: DO NOT USE THE STAGING-LATEST VERSION OF THE DRIVER FOR PRODUCTION
+DISCLAIMER: THE LATEST IMAGE IS CONSTANTLY CHANGING WITH DEVELOPMENT AND CAN BE
+BROKEN AT ANY TIME
+
+This is the absolute cutting edge development Driver, it is intended for testing
+and development only and can have vast differences in
+functionality/behavior/configuration. Use only to try the newest features that
+are not guaranteed to work yet.
```

Comment:
Add a comment as well that the authentication works differently, and assumes that cluster nodes have set up ADC correctly via scopes.

I know it's documented elsewhere but it's a hard problem to track down if you aren't looking for it so better to overcommunicate.

Reply:
Or at least point to the readme.